contrib/utilities/indent

```diff
@@ -67,7 +67,9 @@ find tests include source unit_tests cookbooks benchmarks \( -name '*.cc' -o -na
 tab_to_space()
 {
     f=$1
-    sed -e 's/\t/  /g' $f >$f.tmp
+    # awkward tab replacement because of OSX sed, do not change onless you test it on OSX
+    TAB=$'\t'
+    sed -e "s/$TAB/  /g" $f >$f.tmp
```

Comment:
Does `$'\t'` yield the string expansion of `\t`?

I'll note that that just expands tabs into two spaces. That's not what tabs do -- they expand to a number of spaces necessary to get to the next multiple of 8 characters (or whatever you set the tab width to). This is likely going to lead to some ugly text...

Reply:
> Does `$'\t'` yield the string expansion of `\t`?

It will yield the actual tab character (instead of \ followed by t), see https://stackoverflow.com/a/2623007

> I'll note that that just expands tabs into two spaces.

Correct, but this is what is happening on master already (I think you reviewed the patch: #3035). It probably doesn't look pretty, but the point is to fail on the tester if you try to commit files with tabs in them. This PR just makes it work on OSX.

Reply:
Yes, I thought about that in #3035 where I did think that it looked ugly. Oh well.